src/metabase/driver/sql/query_processor.clj

```diff
@@ -136,30 +136,56 @@
   [_ value]
   (h2x/->integer value))
 
-(defmulti ->float
+(defmulti float-dbtype
+  "Return the name of the floating point type we convert to in this database."
+  {:changelog-test/ignore true :added "0.55.0" :arglists '([driver])}
+  driver/dispatch-on-initialized-driver
+  :hierarchy #'driver/hierarchy)
+
+(defmethod float-dbtype :sql
+  [_driver]
+  "float")
+
+(defmulti cast-float
   "Cast to float."
   {:changelog-test/ignore true :added "0.45.0" :arglists '([driver honeysql-expr])}
```

Comment:
Change the `added` metadata and should it be ignored?

Reply:
:+1: